docs/quickstart.rst

```diff
@@ -57,15 +57,20 @@ deprecation indicators are now available.
     **Shortcut!**
 
     In order to bypass the installation and configuration of *Redis* required by the
-    default storage and cache, use the «in-memory» backend in :file:`myproject.ini`:
+    default storage and cache, use the «in-memory» backend in :file:`development.ini`:
 
     .. code-block:: ini
 
-        # myproject.ini
+        # development.ini
         cliquet.cache_backend = cliquet.cache.memory
         cliquet.storage_backend = cliquet.storage.memory
 
 
+Now is a good time to install the project::
+
+    $ python setup.py develop
```

Comment:
I've also had to install the project in order to make it work, with pcreate and cornice templates, it seems weird when you come from django for instance, but is required in pyramid.

This makes me think we could make a cliquet template for pcreate, [as we did for cornice](https://github.com/mozilla-services/cornice/tree/master/cornice/scaffolds) which doesn't require to copy/paste any code.